stream_alert/rule_processor/rules_engine.py

```diff
@@ -384,22 +383,42 @@ def process(self, input_payload):
                     record_copy = record.copy()
                     record_copy[NORMALIZATION_KEY] = types_result
                     if self._threat_intel:
-                        normalized_records.append(record_copy)
+                        # An copy of payload which includes payload metadata.
+                        # The payload metadata includes log_source, type, service,
+                        # and entity. The meatdata will be returned to along with
```

Comment:
Typo: `The meatdata will be returned to along with...` -- > `The meatdata will be returned along with...`